script.js

```diff
@@ -4,4 +4,219 @@ const mobileMenu = document.querySelector('.mobile-menu');
+
+};
+
+const project5 = {
+  closeButton: 'images/modal-close-button.svg',
+  name: 'Keeping track of hundreds of components',
+  description: 'Multi-Post Stories<br>Gain+Glory',
+  imageMobile: 'images/modal-mobile-background.png',
+  imageDesktop: 'images/modal-desktop-backgroun.png',
+  ubySkill: 'Ruby on rails',
+  cssSkill: 'css',
+  jsSkill: 'JavaScript',
+  htmlSkill: 'html',
+  CodeKit: 'Codekit',
+  gitHub: 'GitHub',
+  boostrap: 'Boostrap',
+  terminal: 'Terminal',
+  codePen: 'Codepen',
+  seeProject: 'See Project',
+  live: 'https://stephnna.github.io/portfolio/',
+  source: 'https://github.com/stephnna/portfolio',
+  modalDescription: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quae modi inventore ex aut voluptatum eaque a repellendus animi dolore sed  temporibus autem amet quod, tempore architecto vitae aspernatur minima aliquid  Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt amet eligendi voluptatum mollitia autem, veritatis dicta doloremque nulla quibusdam,iusto consequatur eum ad temporibus autem amet quod, tempore architecto vitae aspernatur minima aliquid  Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt amet eligendi voluptatum mollitia autem, veritatis dicta doloremque nulla quibusdam, iusto consequatur eum ad deleniti tempora et atque nihil voluptate veniam?',
+
+};
+
+const project6 = {
+  closeButton: 'images/modal-close-button.svg',
+  name: 'Keeping track of hundreds of components',
+  description: 'Multi-Post Stories<br>Gain+Glory',
+  imageMobile: 'images/modal-mobile-background.png',
+  imageDesktop: 'images/modal-desktop-backgroun.png',
+  rubySkill: 'Ruby on rails',
+  cssSkill: 'css',
+  jsSkill: 'JavaScript',
+  htmlSkill: 'html',
+  CodeKit: 'Codekit',
+  gitHub: 'GitHub',
+  boostrap: 'Boostrap',
+  terminal: 'Terminal',
+  codePen: 'Codepen',
+  seeProject: 'See Project',
+  live: 'https://stephnna.github.io/portfolio/',
+  source: 'https://github.com/stephnna/portfolio',
+  modalDescription: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quae modi inventore ex aut voluptatum eaque a repellendus animi dolore sed  temporibus autem amet quod, tempore architecto vitae aspernatur minima aliquid  Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt amet eligendi voluptatum mollitia autem, veritatis dicta doloremque nulla quibusdam,iusto consequatur eum ad temporibus autem amet quod, tempore architecto vitae aspernatur minima aliquid  Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt amet eligendi voluptatum mollitia autem, veritatis dicta doloremque nulla quibusdam, iusto consequatur eum ad deleniti tempora et atque nihil voluptate veniam?',
+
+};
+
```

Comment:
Also, your object should contain mostly the following 
- Name
- Description
- image
- Technologies - array
- Live link 
- Github link

Reply:
I have adjusted the close button so that it clears from the background image.

Adding the social details of all the contributors is not part of the microverse requirement for approving this project as stated here https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md

Moreover, GitHub automatically list all the contributors on the right side of the projects. Meaning that if someone is interested in checking them out, he can click on  them which will open up their Github page where he will find much more information about the person and not just the socials details only.

I have also implemented some of your optional changes